src/page-multi-facility/ScenarioLibraryModal.tsx

```diff
@@ -114,7 +114,7 @@ const IconRecidviz = styled.img`
 
 const ScenarioLibraryModal: React.FC<Props> = ({ trigger }) => {
   const [modalOpen, setModalOpen] = useState(false);
-  const [_, dispatchScenarioUpdate] = useScenario();
+  const [, dispatchScenarioUpdate] = useScenario();
```

Comment:
Apparently the linter isn't smart enough to consider variables names in a destructured array, but apparently you can also just not include the unused variable.